### DISCUSSION TITLE
Cannot read Current Track URL

### DISCUSSION THREAD

**Discussion by GaryCraft** [Androz2091/discord-player#688] [Q&A]
**Description**
The bug appears every time I try to read out the URL of the current playing track

**To Reproduce**

1. Create a queue
2. Add a track
3. Try to console.log() current track URL

**Expected behavior**
queue.current.url logs the track URL.

**Please complete the following information:**
 - Node Version: [16.6.1]
 - Discord Player Version: [5.0.2]
 - Discord.js Version: [13.0.1]

**Additional context**

**Comment by twlite** [comment]
@GaryCraft can you explain more?

**Comment by GaryCraft** [comment]
As described above I seem to have an Issue to logging \<Queue\>.current.url it shows as undefined

**Comment by twlite** [comment]
@GaryCraft Hello, I just ran a test on the **[example bot](https://github.com/Androz2091/discord-player/blob/master/example/music-bot/index.js)** and I see no problem with `queue.current.url`
